internal/cmd/logs/util.go

```diff
@@ -34,7 +34,38 @@ func (e events) Less(i, j int) bool {
 	return ptr.ToTime(e[i].Timestamp).Unix() < ptr.ToTime(e[j].Timestamp).Unix()
 }
 
-func printLogs(logs events) {
+func reduceLogsByLength(logsRange int, logs *events) *events {
```

Comment:
I think `reduceLogsToLength` would be a better name. "By length" to me implies that you will subtract that much from it.

Reply:
agreed, will change it to that